pkg/upstream/http_test.go

```diff
@@ -214,7 +214,7 @@ var _ = Describe("HTTP Upstream Suite", func() {
 					URL:    "http://example.localhost/withSignature",
 					Header: map[string][]string{
 						gapAuth:      {""},
-						gapSignature: {"sha256 osMWI8Rr0Zr5HgNq6wakrgJITVJQMmFN1fXCesrqrmM="},
+						gapSignature: {"sha256 cbfsdaehGMvhwEc/qd4QJb1J41cmNRw9fTBUzFFmnKw="},
```

Comment:
Since we removed the GAP-Signature header change, these likely need to stay unchanged as well, right?

Reply:
You are right I forgot to push the reverted test files!